### PR TITLE
Add gocql test

### PR DESCRIPTION
**Pull request by HarshDaryani896**

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -28,6 +28,10 @@ SEQU_RUN=$?
 YBDB_IMAGE_PATH=$YBDB_IMAGE_PATH bash ./gorm/start.sh
 GORM_RUN=$?
 
+# 3. GOCQL
+YBDB_IMAGE_PATH=$YBDB_IMAGE_PATH bash ./gocql/start.sh
+GOCQL_RUN=$?
```
Comment (ashetkar) [inline start.sh:33]: Include this in the EXIT_CODE below.

File: gocql/tear-down.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Cleanup any processes that are still running/hung
+. ./process-cleanup.sh
```
Comment (ashetkar) [inline gocql/tear-down.sh]: I don't see any process-cleanup.sh file. Can remove this if not required.

File: gocql/run-app.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+rm -rf $WORKING_DIR/gocql
+git clone git@github.com:yugabyte/gocql.git -b partition_aware_policy $WORKING_DIR/gocql
+pushd $WORKING_DIR/gocql
```
Comment (ashetkar) [inline gocql/run-app.sh:6]: popd missing for this?

File: gocql/run-app.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+go test -timeout 30s -run ^TestGetKey$ github.com/yugabyte/gocql > $ARTIFACT_PATH/gocql-TestGetKey-output.txt
+REST_PID2=`echo $!`
+
+# Allow some time for server init
+sleep 10
+
+#grep "ok" $ARTIFACT_PATH/gocql-TestHostRouting-output.txt
+grep "ok" $ARTIFACT_PATH/gocql-TestGetKey-output.txt
+printf "Verified example output.\n"
+
```
Comment (ashetkar) [inline gocql/run-app.sh:19]: Does it log something more than just 'ok'? Feel it's too short a string to check for success.

Comment (HarshDaryani896) [reply gocql/run-app.sh:19]: It just prints ok and the time it took to run the test. for eg:  'ok      github.com/yugabyte/gocql       0.696s'

**Review by ashetkar — APPROVED** [review]

**Pull request merged**